src/services/ui/src/utils/Poller/seaStatusPoller.ts

```diff
@@ -0,0 +1,39 @@
+import { getItem } from "@/api/useGetItem";
+import { DataPoller } from "./DataPoller";
+import { PackageCheck } from "shared-utils";
+
+/** These are checks that simply can't be added into the packagecheck logic.
+ * A prime example being the ability to check for the existance of the data
+ * coming back. This won't work in something like PackageCheck because it is
+ * already expecting data
+ */
+export const pollerSpecificChecks = (
+  data: Awaited<ReturnType<typeof getItem>> | undefined | null,
+) => {
+  const recordExists = !!data;
```

Comment:
any reason we can't just measure inline with `!!data` where this is getting used?

Reply:
I've thought about my design for this and didn't love it. I have idea though where if you don't pass anything to the onPoll where it would just check to see for the existance of the data and come back true.

Reply:
what are you specifically trying to accomplish with it? i looked at it and it seems like this feeds into the poller, but this is ultimately just returning a double bang on the parameter already given, so you have access to this boolean logic in the next scope up anyway